python/lsst/afw/table/Base.i

```diff
@@ -122,6 +122,7 @@ template <> struct NumpyTraits<lsst::afw::geom::Angle> : public NumpyTraits<doub
 %pythoncode %{
 from . import _syntax
 import astropy.units
+from past.builtins import basestring
```

Comment:
In this particular case I don't want the futurized `str` to contaminate the `aliases` table.